ReStyle-Frontend/src/app/components/item-card/item-card.component.css

```diff
@@ -0,0 +1,103 @@
+/* ----------------------------------------------
+ * Generated by Animista on 2019-5-5 13:37:19
+ * w: http://animista.net, t: @cssanimista
+ * ---------------------------------------------- */
+
+/**
+ * ----------------------------------------
+ * animation slide-out-left
+ * ----------------------------------------
+ */
+ @-webkit-keyframes slide-out-left {
+  0% {
+    -webkit-transform: translateX(0);
+            transform: translateX(0);
+    opacity: 1;
+  }
+  100% {
+    -webkit-transform: translateX(-1000px);
+            transform: translateX(-1000px);
+    opacity: 0;
+  }
+}
+@keyframes slide-out-left {
+  0% {
+    -webkit-transform: translateX(0);
+            transform: translateX(0);
+    opacity: 1;
+  }
+  100% {
+    -webkit-transform: translateX(-1000px);
+            transform: translateX(-1000px);
+    opacity: 0;
+  }
+}
+
+
+/* ----------------------------------------------
+ * Generated by Animista on 2019-5-3 14:44:13
+ * w: http://animista.net, t: @cssanimista
+ * ---------------------------------------------- */
+
+/**
+ * ----------------------------------------
+ * animation slide-out-right
+ * ----------------------------------------
+ */
+ @-webkit-keyframes slide-out-right {
```

Comment:
Something about the slide out right animation creates a horizontal scroll bar. This needs to be fixed. The left slide out works fine

Reply:
Yes, I see that too. I'll look into it.